src/Sinch/Conversation/Hooks/EventDelivery.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
```

Comment:
All events related files/classes are suffixed with `Event`, but EventDelivery, EventDeliveryAllOfEventDeliveryReport, EventInBound and EventInBoundAllOfEvent are exceptions to this rule
Is there a reason ?

Reply:
Generator 🤷 It takes `x-webhooks.{name}` as a class name, it's the same in oas file https://github.com/sinch/sinch-sdk-oas-generator/blob/473bafeb76f682f4f6d51ee3da63c057f907c130/openapi-contracts/conversation-oas-1.0.yaml#L90